backend/game_server/README.md

```diff
@@ -0,0 +1,24 @@
+# GameServer
```

Comment:
I would use more specific main namespace name,
e.g. `BarnacleServer`.
so general names are better for general use libraries, this is very specific project.